include/sentry.h

```diff
@@ -1493,6 +1493,12 @@ SENTRY_API void sentry_set_fingerprint_n(
  */
 SENTRY_API void sentry_remove_fingerprint(void);
 
+/**
+ * Sets the trace ID.
+ */
```

Comment:
Since this is now part of the public API and could be used by all clients, not only downstream SDKs, it would be great to have some context here. Just basic information on what it is used for, what clients shouldn't use it for, and a link to conceptual docs (not developer docs), so clients understand the usage scenario.